tests/unit/karma.conf.js

```diff
@@ -17,6 +17,7 @@ module.exports = function(config) {
             'media/js/base/mozilla-accordion-gatrack.js',
             'media/js/base/mozilla-accordion.js',
             'media/js/base/mozilla-client.js',
+            'media/js/base/mozilla-cookie-helper.js', // mozilla-traffic-cop.js
```

Comment:
Rather than rely on this lib being present in the Karma bundle for the `mozilla-traffic-cop.js` tests to pass, I'd suggest removing it here and then mocking the `mozilla-cookie-helper` functions to return expected values for each test. This separates the dependency for testing and also means we're not creating things like cookies in the test runner.